var/spack/repos/builtin/packages/dust/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dust(Package):
+    """du + rust = dust. Like du but more intuitive."""
+
+    homepage = "https://github.com/bootandy/dust"
+    url = "https://github.com/bootandy/dust/archive/v0.7.5.tar.gz"
+
+    maintainers = ["fangohr"]
+
+    version(
+        "0.7.5",
+        sha256="f892aaf7a0a7852e12d01b2ced6c2484fb6dc5fe7562abdf0c44a2d08aa52618",
+    )
+
+    depends_on("rust")
+
+    sanity_check_is_file = [join_path("bin", "dust")]
```

Comment:
I don't see this used anywhere, is it needed for something? If not, can you remove it?

Reply:
The line checks if the file `.../bin/dust` is created when the package is installed. If it isn't, it raises an exception. 

The use of these sanity checks is mentioned in the packaging guide (https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-sanity-checks).